File: Starters/Starters 42/EQUALDIST/Solution.c
#include <stdio.h>

int main() {
    int a,b,i,t;
    scanf("%d",&t);
    for(i=0;i<t;i++){
        scanf("%d%d",&a,&b);
        if((a+b)%2 == 0)
        printf("YES\n");
        else printf("NO\n");
    }
	return 0;
}

File: Starters/Starters 42/EQUALDIST/Solution.go
package main
import "fmt"

func main() {
  var a,b,t int
  fmt.Scanln(&t)
  for i:=0;i<t;i++ {
    fmt.Scan(&a,&b)
    if (a+b)%2 == 0{
      fmt.Println("YES")
    } else {
      fmt.Println("NO")
    }
}
